base/src/ast.rs

```diff
@@ -260,16 +260,18 @@ impl<Id> LExpr<Id>
     /// Returns the an approximation of the span of the expression
     pub fn span<'a>(&self, env: &(DisplayEnv<Ident = Id> + 'a)) -> Span {
         use self::Expr::*;
+
+        let start = self.location.absolute;
```

Comment:
I suppose I already mentioned this on gitter but to document it here as well.

`absolute` is never set to a proper value in the parser so it can't be used :(